custom/icds_reports/static/icds_reports/js/spec/download.directive.spec.js

```diff
@@ -289,7 +289,7 @@ describe('Download Directive', function () {
         }));
 
         it('tests that all users have access to ISSNIP monthly register', function () {
-            var expected = 10;
+            var expected = 11;
```

Comment:
In this test cases I think FF is off so should not it be let 10? or M I missing somethign? @challabeehyv

Reply:
it has feature flag enabled. so should be 11